src/custom/hooks/useSwapCallback.ts

```diff
@@ -50,13 +50,7 @@ function getSummary(params: PostOrderParams): string {
   }
 }
 
-function signOrder(unsignedOrder: UnsignedOrder): string {
-  console.log('TODO: Sign order', unsignedOrder)
-  // TODO: Mocked, next PR
-  return '0xd6741f8031e7a7ea70f1b6c14a71e9d0d7d5aae54c157d8368ad4cabd7279a363955fa169a634469da01010bfefbacc2c1e2e7aaeb0c42049192e6359ed572281b'
-}
-
-async function postOrderApi(params: PostOrderParams, signature: string): Promise<OrderID> {
+async function _postOrderApi(params: PostOrderParams, signature: string): Promise<OrderID> {
```

Comment:
`params` should be of `OrderCreation type` if you gonna post them and not preprocess
you gonna preprocess?

Reply:
planning to do this in new PR, with the impl, but I'll do it now...